### PR TITLE
Add support for Ruby 2.1

### LINKED ISSUE #70: Add Ruby 2.1 support
I'd like to test a gem on Ruby 2.1 to ensure legacy compatibility. I understand that this Ruby version is not supported upstream anymore (as is Ruby 2.2). However, since Ruby 2.1 was shipped with some older operating systems (notably Debian 8 Jessie), I'd still like to support it with my code if possible, and obviously need to test this.

In fact, adding support to ruby/setup-ruby is rather straight forward. I have a WIP branch for that at [meineerde/setup-ruby@feature/ruby-2.1](https://github.com/meineerde/setup-ruby/tree/feature/ruby-2.1). For Windows builds, this alone is enough. For macOS and Ubuntu, there would still need to be builds generated in [ruby/ruby-builder](https://github.com/ruby/ruby-builder).

Is this something you would be interested in?

### PR DESCRIPTION
**Pull request by meineerde**
This adds support for Ruby 2.1. This resolves #70 

What is still missing here is support in the [ruby/ruby-builder](https://github.com/ruby/ruby-builder) repo to also build Ruby 2.1 for Ubuntu and macOS.

### CODE REVIEW

File: ruby-builder-versions.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export function getVersions(platform) {
   const versions = {
     "ruby": [
+      "2.1.10",
```
Comment (eregon) [inline ruby-builder-versions.js:4]: Should be 2.1.9 as mentioned on the issue

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ This action currently supports these versions of MRI, JRuby and TruffleRuby:
 
 | Interpreter | Versions |
 | ----------- | -------- |
-| Ruby | 2.2, 2.3.0 - 2.3.8, 2.4.0 - 2.4.10, 2.5.0 - 2.5.8, 2.6.0 - 2.6.6, 2.7.1, head, debug, mingw, mswin |
+| Ruby | 2.1, 2.2, 2.3.0 - 2.3.8, 2.4.0 - 2.4.10, 2.5.0 - 2.5.8, 2.6.0 - 2.6.6, 2.7.1, head, debug, mingw, mswin |
```
Comment (eregon) [inline README.md:17]: Let's write `2.1.9` here to make it clear only that release is available

File: index.js [hunk 3]
```diff
@@ -191,8 +191,8 @@ async function installBundler(bundlerVersionInput, platform, rubyPrefix, engine,
     throw new Error(`Cannot parse bundler input: ${bundlerVersion}`)
   }
 
-  if (rubyVersion.startsWith('2.2')) {
-    console.log('Bundler 2 requires Ruby 2.3+, using Bundler 1 on Ruby 2.2')
+  if (rubyVersion.match(/^2\.[12]/)) {
+    console.log('Bundler 2 requires Ruby 2.3+, using Bundler 1 on Ruby 2.1 and 2.2')
```
Comment (eregon) [inline index.js:195]: Can you simplify to `on Ruby <= 2.2`?

Comment (meineerde) [reply index.js]: Updated.

**Comment by meineerde** [comment]
I have added new commits with your suggestions. If you are satisfied, I can squash all the commits.

File: README.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ This action currently supports these versions of MRI, JRuby and TruffleRuby:
 
 | Interpreter | Versions |
 | ----------- | -------- |
-| Ruby | 2.2, 2.3.0 - 2.3.8, 2.4.0 - 2.4.10, 2.5.0 - 2.5.8, 2.6.0 - 2.6.6, 2.7.1, head, debug, mingw, mswin |
+| Ruby | 2.1.9, 2.2.10, 2.3.0 - 2.3.8, 2.4.0 - 2.4.10, 2.5.0 - 2.5.8, 2.6.0 - 2.6.6, 2.7.1, head, debug, mingw, mswin |
```
Comment (eregon) [inline README.md:17]: For 2.2 it's 2.2.6 on Windows and 2.2.10 otherwise (see below), unfortunately.
So I guess `2.2` in the description is best.

Comment (meineerde) [reply README.md:17]: Hrmpf... You are right of course :)

**Comment by meineerde** [comment]
Okay, I think this is it. Thanks for the review! If you'd like you, you may just squash the whole enchilada on your own.

**Pull request merged**